source/cost-optimization.txt

```diff
@@ -7,13 +7,41 @@ Cost Optimization
 
 .. default-domain:: mongodb
 
-Use the following resources to learn about cost optimization in
-{+service+}:
+The Cost Optimization pillar of the {+service+} {+waf+} includes
+features that help you control database costs in |service-fullname|.
 
-- :ref:`Cost-Saving Configurations <arch-center-cost-saving-config>`
-- :ref:`Billing Data <arch-center-billing-data>`
 
 
+
+Use the following {+atlas-arch-center+} resources to learn about cost
+optimization in {+service+}:
+
+.. card-group::
+   :columns: 2
+   :style: extra-compact
+
+   .. card::
+      :headline: Cost-Saving Configurations
+      :url: https://mongodb.com/docs/tbd/cost-saving-config/
```

Comment:
@sarahsimpers, please search for tbd in all topics. I will replace these, of course, with the full paths once we have them. Thank you!